lib/src/tooltip_action.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+
+class ToolTipAction extends StatelessWidget {
+  const ToolTipAction({
+    Key? key,
+    required this.actionOne,
+    required this.actionTwo,
+    this.actionPadding,
+  }) : super(key: key);
+
+  /// Define configuration of first action
+  final ActionWidgetConfig actionOne;
+
+  /// Define configuration of second action
+  final ActionWidgetConfig actionTwo;
+
+  /// Define padding of action widget
+  final EdgeInsets? actionPadding;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: actionPadding ?? EdgeInsets.zero,
+      child: Wrap(
+        alignment: WrapAlignment.spaceBetween,
+        crossAxisAlignment: WrapCrossAlignment.center,
+        children: [
+          getActionWidget(actionOne),
```

Comment:
Can we use List<ActionWidgetConfig> here? I guess the user gets more feasibility to add more than two actions.